resin/models/device.py

```diff
@@ -17,6 +18,8 @@ class Device(object):
 
     """
 
+    RESIN_CREATION_YEAR = 2013
```

Comment:
PineJS returns '1970-01-01' for null dates, instead of null -- it's a bug. So, some legacy code used ``year < 2013`` to filter these dates (since it's impossible to get anything dating before that given that Resin is younger that that!) -- and this code was carried over to other places :)